benchmarks/benchmarks.py

```diff
@@ -23,10 +27,41 @@ def time_init(self):
 class InitScmBench(BaseBench):
     def setup(self):
         super().setup()
-        from git import Repo
 
-        git = Repo.init(self.test_directory.name)
+        git = GitRepo.init(self.test_directory.name)
         git.close()
 
     def time_init(self):
         main(["init"])
+
+
+class AITABench(BaseBench):
```

Comment:
Maybe let's create separate files for different benchmarks? Should be easier to manage. Or does that not work with asv?

Reply:
Should be fine AFAIK